lib/document_viewer/secrets_provider.ex

```diff
@@ -0,0 +1,103 @@
+defmodule DocumentViewer.SecretsProvider do
```

Comment:
If this is an ECS app, you don't need this. You can configure the ECS task to load the secrets in as environment variables.